diff --git a/node_modules/@notionhq/client/build/src/api-endpoints.d.ts b/node_modules/@notionhq/client/build/src/api-endpoints.d.ts
index 3007c4c..95dbc88 100644
--- a/node_modules/@notionhq/client/build/src/api-endpoints.d.ts
+++ b/node_modules/@notionhq/client/build/src/api-endpoints.d.ts
@@ -317,7 +317,7 @@ export type PageObjectResponse = {
     archived: boolean;
     url: string;
 };
-export type PartialPageObjectResponse = {
+export type PartialPageObjectResponse = PageObjectResponse & {
     object: "page";
     id: string;
 };
